src/main/no/wtw/android/dijkstra/DijkstraAlgorithm.java

```diff
@@ -127,6 +130,9 @@ private int getDistance(Vertex from, Vertex to) throws PathNotFoundException {
     public int getShortestPathLength(Vertex from, Vertex to) throws PathNotFoundException {
         if (from == null || to == null)
             throw new IllegalArgumentException("Source and destination vertices can not be null");
+        if(getPath(to) == null) {
```

Comment:
Kan getPath(to) noensinne bli null?
getPath er satt til å kaste PathNotFoundException selv.